### PR TITLE
feat: remove metrika goals

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

**Comment by NikitaCG** [comment]
you need create PR in `next` brach

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@doc-tools/transform": "^3.11.0",
     "@gravity-ui/eslint-config": "^3.1.1",
-    "@gravity-ui/page-constructor": "^4.46.0",
+    "@gravity-ui/page-constructor": "^4.51.2-alpha.0",
```
Comment (NikitaCG) [inline package.json:85]: merge after page-constructor5

**Review by NikitaCG — APPROVED** [review]

File: src/blocks/Meta/Meta.tsx [hunk 2]
```diff
@@ -80,7 +77,8 @@ export const Meta = (props: MetaProps) => {
                     postId={id}
                     date={date}
                     readingTime={readingTime}
-                    metrikaGoals={metrikaGoals}
+                    analyticsEventsContainer={analyticsEventsContainer}
+                    // metrikaGoals={metrikaGoals}
```
Comment (NikitaCG) [inline src/blocks/Meta/Meta.tsx:81]: and this

**Review by NikitaCG — CHANGES_REQUESTED** [review]

**Review by NikitaCG — APPROVED** [review]

File: src/components/PostCard/PostCard.tsx [hunk 3]
```diff
@@ -81,7 +78,8 @@ export const PostCard = ({
     return (
         <CardBase
             url={url}
-            metrikaGoals={metrikaGoals}
+            // metrikaGoals={metrikaGoals}
```
[the remark this answers is not in the document]
Comment (niktverd) [reply src/components/PostCard/PostCard.tsx]: sure

**Pull request merged**